applications/nrf_desktop/doc/board.rst

```diff
@@ -26,9 +26,10 @@ Configuration
 The module uses Zephyr's :ref:`zephyr:gpio_api` driver to set the pin state.
 For this reason, set the :kconfig:`CONFIG_GPIO` option.
 
-For every configuration, you must define the :file:`port_state_def.h` file in the board-specific directory in the application configuration directory.
+For every configuration, you must define the configuration file specified by the :kconfig:`CONFIG_DESKTOP_BOARD_DEF_PATH` option.
+The configuration file should be placed in the board-specific directory in the application configuration directory.
```

Comment:
Mirror edits from previous comments. My understanding is that you need to specify the file in this Kconfig at this stage. The current doc edit suggests it has been already specified by the user.